README.md

```diff
@@ -15,41 +15,82 @@ Yes! *She's Coding* is build by the community, for the community. We currently h
 ### Installation
 
 + Install [Homebrew](http://brew.sh/) (OSX):
-  ```
-  ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
-  ```
-
-+ Install [rvm](https://rvm.io/):
-  ```
-  \curl -sSL https://get.rvm.io | bash -s stable
-  ```
 
-+ Install [Ruby](https://www.ruby-lang.org/en/):
-  ```
-  rvm install 2.2.1
-  ```
-
-+ Select this version for use:
-  ```
-  rvm use 2.2.1
   ```
+  $ ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+  ```
+
++ Set the [Ruby](https://www.ruby-lang.org/en/) version to 2.3.1. In order to be able to pick the ruby version or to download ruby, you will need to use Ruby version manager. There are 2 ruby version management tools: [rvm](https://rvm.io/) and [rbenv](https://github.com/rbenv/rbenv).
+
```

Comment:
Could you change the 'you will need to use Ruby version manager' to 'we recommend using a ruby version manager'? Thank you!